hack/functests.sh

```diff
@@ -26,4 +26,4 @@ if [[ ${TARGET} == openshift* ]]; then
     oc=${kubectl}
 fi
 export KUBEVIRT_PROVIDER=$KUBEVIRT_PROVIDER
-${TESTS_OUT_DIR}/tests.test -kubeconfig=${kubeconfig} ${FUNC_TEST_ARGS}
+${TESTS_OUT_DIR}/tests.test -kubeconfig=$(_kubevirtci/cluster-up/kubeconfig.sh) ${FUNC_TEST_ARGS}
```

Comment:
Instead of calling this implicitly created script, use `cluster/` interface, like this one https://github.com/nmstate/kubernetes-nmstate/blob/master/cluster/kubeconfig.sh